include/deal.II/fe/fe_tools_extrapolate.templates.h

```diff
@@ -697,7 +704,18 @@ namespace FETools
         }
     }
 
+    template <typename T>
+    inline constexpr bool is_la_vector = std::false_type{};
+
+    template <typename... Ts>
+    inline constexpr bool
+      is_la_vector<LinearAlgebra::distributed::Vector<Ts...>> =
+        std::true_type{};
 
+    template <typename... Ts>
+    inline constexpr bool
+      is_la_vector<LinearAlgebra::distributed::BlockVector<Ts...>> =
+        std::true_type{};
```

Comment:
You are initializing `bool` variables here, so I think this would be clearer:
```suggestion
    template <typename T>
    inline constexpr bool is_la_vector = false;

    template <typename... Ts>
    inline constexpr bool
      is_la_vector<LinearAlgebra::distributed::Vector<Ts...>> =
        true;

    template <typename... Ts>
    inline constexpr bool
      is_la_vector<LinearAlgebra::distributed::BlockVector<Ts...>> =
        true;
```
I think it might also be nice to spell out the possible template arguments of the Vector/BlockVector class.